src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: normal;
+  font-style: normal;
+  src: url("../src/fonts/Avenir.ttc") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: normal;
+  font-style: normal;
+  src: url("../src/fonts/Avenir-Book.ttf") format("opentype");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: bold;
+  font-style: normal;
+  src: url("../src/fonts/Avenir-Heavy.ttf") format("opentype");
+}
+
+form {
+  margin: 20px 0;
+}
+
+.search-big__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding: 20px 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff no-repeat 25px center/19px url("../src/images/Search.svg");
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.search-big__input:hover {
+  background: #fff no-repeat 25px center/19px url("../src/images/Search.svg");
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-big__input:focus {
+  background: no-repeat 25px center url("../src/images/Search.svg"), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-weight: bold;
+  outline: none;
+}
+
+.search-big__input:focus::placeholder {
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.search-small__input {
+  width: 100%;
+  height: 42px;
+  padding: 11px 33px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff no-repeat 12px 14px/11px url("../src/images/Search.svg");
+  font-family: Avenir-Book, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
```

Comment:
Do not repeat your self
![2023-01-27_18h31_45](https://user-images.githubusercontent.com/61904995/215140942-122ccac0-b024-4e6e-817d-95d0aad4053e.png)